tools/ovc/unit_tests/ovc/utils/args_to_string_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
do we have any tests for `ovc` tool? And `mo` tool?

Reply:
Added `ovc` tool test tests/layer_tests/mo_python_api_tests/test_ovc_cli_tool.py
`mo` tool currently tested in `mo_args` tests and in some tests from MO Python API group (LegacyExtTest, TestSubprocessMoConvert)